### DISCUSSION TITLE
Fronius Primo Inverter documentation

### DISCUSSION THREAD

**Discussion by petebitt99** [amberframework/amber#1360] [Q&A]
Would anyone have access to the technical manual for this inverter. I am trying to do what many of you are doing and send it write commands to adjust its power output.  So I need to know what registers to read and write.
Maybe someone knows just the info to do that so if you can share that would be appreciated.

**Comment by crimson-knight** [comment]
Sorry but I think you have the wrong repo. You appear to be looking for technical documentation for a solar inverter and this project is for an open source web framework called Amber.
